reactor-netty5-http/src/test/java/reactor/netty5/http/client/HttpClientProxyTest.java

```diff
@@ -158,7 +158,7 @@ void testIssue804(Hoverfly hoverfly) {
 				            false))
 				    .expectNextMatches(t ->
 				           t.getT2().contains("Hoverfly") &&
-				                "NOT FOUND".equals(t.getT2().get("Logging-Handler")) &&
+				                "NOT FOUND".equals(t.getT2().get("Logging-Handler").toString()) &&
```

Comment:
This may produce `NullPointerException`

Reply:
I have added a "getHeader" helpers method in BaseHttpTest.java, which avoids to do the toString() everywhere.
There are many other places where toString() was used, so I also fixed with the getHeader helper, please check.